### DISCUSSION TITLE
Highlight finished jobs.

### DISCUSSION THREAD

**Discussion by bradleymaustin** [UnchartedBull/OctoDash#2375] [Q&A] [answered]
Is there anyway to have completed prints highlighted on the jobs screen like on the web view? Makes it hard to remember what I last printed when doing large amounts of long prints.

**Comment by jneilliii** [comment, marked as answer]
This has actually already been merged into the code base, but hasn't been released as a version yet. If you want to try it out you can grab the latest deb file from this link.

https://github.com/jneilliii/OctoDash/actions/runs/1375690056

Download the artifact at the bottom and extract on your machine. Then transfer the octodash_2.2.0_armv7l.deb file over to your pi using filezilla/winscp or other means, then install over your existing install using `sudo dpkg -i octodash_2.2.0_armv7l.deb` via SSH. Once installed restart OctoDsh with `sudo service getty@tty1 restart` and when it loads back up it should prompt for settings changes in latest version. If this fails for whatever reason you may have to delete the config.json file and redo the initial setup wizard.
